convert2rhel/actions/pre_ponr_changes/subscription.py

```diff
@@ -117,7 +135,10 @@ def run(self):
             self.set_result(
                 level="ERROR",
                 id="MISSING_SUBSCRIPTION_MANAGER_BINARY",
-                message="Failed to execute command: %s" % e,
+                title="Missing subscription-manager binary",
+                description="There is a missing subscription-manager binary",
+                diagnosis="Failed to execute command: %s" % e,
+                remediation="On Centos execute 'yum install subscription-manager'",
```

Comment:
Not sure if we should assume that the system is CentOS always. Maybe we should do something like: 

```suggestion
                remediation="On %s execute 'yum install subscription-manager'" % system_info.name,
```

I don't remember right now if subscription-manager is included in the base repos for OL, if it is, then I guess we can do this change like I suggested.

Reply:
That remediation is specific to CentOS

Reply:
sure, but there is nothing in the code that prevents this to being output to another system. A user from OL/Alma/Rocky would be confused seeing this.

Reply:
Good point, Should we just remove the remediation then?

Reply:
Can make it conditional, but for now I agree on removing